README.md

```diff
@@ -64,7 +70,8 @@ This project is covered under the Apache 2.0 license. You can read the license [
 
 =======
 
-[notation-releases]:      https://github.com/shizhMSFT/notation/releases/tag/v0.5.0
+[notation-releases]:      https://github.com/notaryproject/notation/releases/tag/v0.7.0-alpha.1
```

Comment:
Should we point to this specific notation release, or to the generic release folder https://github.com/notaryproject/notation/releases to keep it compatible with future churns?

Reply:
I had a similar thought. Future churns will have commands that reference new releases. So, I kept the link to the specific release to assure folks don't get broken between releases.